web/handlers.go

```diff
@@ -230,8 +230,9 @@ func (h Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 
 		// Set content security policy. This is also specified in the root.html of the webapp in a meta tag.
 		w.Header().Set("Content-Security-Policy", fmt.Sprintf(
-			"frame-ancestors 'self'; script-src 'self' *.riffplatform.com *.riffedu.com *.riffai.cloud cdn.rudderlabs.com%s%s%s%s",
-			" *.googletagmanager.com 'sha256-K4FCg61IQARpz2f8JyoJsmQfYJOCjZGl0TkKzyDxppE='",
+			"frame-ancestors 'self'; script-src 'self' *.riffplatform.com *.riffedu.com *.riffai.cloud cdn.rudderlabs.com%s%s%s%s%s",
+			" https://cdn.userway.org/ 'sha256-SW4mlJLRxlBpckfBUvfFo6QJIMVl74UJsWvLxoUiW28=' ",
+			"*.googletagmanager.com 'sha256-K4FCg61IQARpz2f8JyoJsmQfYJOCjZGl0TkKzyDxppE='",
```

Comment:
One more change. If google analytics or userway is removed, you shouldn't have to edit the other one. So they should both start with a space and not end with a space, instead of the 1st one having a space before and after and the 2nd one no spaces
```suggestion
			" https://cdn.userway.org/ 'sha256-SW4mlJLRxlBpckfBUvfFo6QJIMVl74UJsWvLxoUiW28='",
			" *.googletagmanager.com 'sha256-K4FCg61IQARpz2f8JyoJsmQfYJOCjZGl0TkKzyDxppE='",
```